qb/select.go

```diff
@@ -163,3 +163,44 @@ func (b *SelectBuilder) AllowFiltering() *SelectBuilder {
 	b.allowFiltering = true
 	return b
 }
+
+// Func sets aggregation function for select clause on the query.
+func (b *SelectBuilder) Func(name, column string) *SelectBuilder {
```

Comment:
I'd not export that function, if you want sth custom you can use Columns directly. Func is sort of a name clash with Func struct.

Also can we put this function bellow the others?

Reply:
Done.